routes/api.php

```diff
@@ -14,6 +14,5 @@
 |
 */
 
-Route::middleware('auth:api')->get('/user', function (Request $request) {
-    return $request->user();
-});
+Route::post('/charge-card', 'PaymentContoller@chargeCard');
```

Comment:
Typo on controller name.

Reply:
Correction has been made on the controller name